File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.java

class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        HashMap<Integer,Integer> map = new HashMap<>();
        for(int n : nums){
                if(map.containsKey(n)){
                    map.put(n,map.get(n)+1);
                }
                map.putIfAbsent(n,1);
        }
        System.out.println(map);

        List<Map.Entry<Integer,Integer>> entries = new ArrayList<>();
        for(Map.Entry<Integer,Integer> entry :map.entrySet()){
                entries.add(entry);
        }

        entries.sort((a,b)->b.getValue()-a.getValue());
        System.out.println(entries);

        int[] output = new int[k];
        for(int i=0; i<output.length;i++){
            output[i] = entries.get(i).getKey();
        }

        return output;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if len(nums)<=0 : 
            return 0
        arr=[]
        lc=1
        nums.sort()
        for i in range(1,len(nums)):
            if nums[i]==nums[i-1]:
                pass
            elif nums[i]-1 == nums[i-1]:
                lc+=1
            else:
                arr.append(lc)
                lc=1
        arr.append(lc)
        arr.sort()
        return arr[len(arr)-1]

            